paasta_tools/cli/cmds/metastatus.py

```diff
@@ -171,6 +173,10 @@ def print_cluster_status(
     )
 
     paasta_print("Cluster: %s" % cluster)
+    try:
+        paasta_print("Mesos leader: %s" % get_mesos_leader())
+    except PaastaNotConfiguredError:
```

Comment:
Do we need this? If so, should we print out something rather than masking the error?

Reply:
This error only happens on a fake testcase which didn't mock a real paasta service. I catch this error just to make that test case pass. Paasta should never be not configured in real life obviously.

Reply:
Gotcha. In this case, could you fix this by mocking at test case instead?

Reply:
Ok. I'll try. I am not familiar with mocking so this might take a few hours.